app/assets/scss/_brief_page.scss

```diff
@@ -0,0 +1,7 @@
+#incomplete-applications, #completed-applications {
+    width: 30%;
+
+    @include govuk-media-query($until: tablet) {
```

Comment:
Nice 👍 

This is a question more for my own curiosity than anything else, but what is the difference between using `$until: tablet` like you've done here and `$from: tablet` like the summary list does?

Reply:
My brain is mush today!

$from: tablet is better in this case, as the width is 100% by default anyway.

And generally $from would be better because that would usually indicate a mobile-first design.